### PR TITLE
Create comagic.json

### PR DESCRIPTION
**Pull request by dred-art**

### CODE REVIEW

**Comment by dred-art** [comment]
I signed it

File: организации/comagic.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+  {
```
Comment (pfrisella) [inline организации/comagic.json]: update the directory name to 'organizations'

Comment (dred-art) [reply организации/comagic.json]: Thank. I can not change the directory. Have to re-create?

**Comment by dred-art** [comment]
I signed it!

**Comment by dred-art** [comment]
@googlebot I signed it!

File: организации/comagic.json (hunk 1 above)
Comment (dred-art) [reply организации/comagic.json]: Changed the directory

**Pull request merged**